rancher/v2.0/en/quick-start-guide/index.md

```diff
@@ -11,86 +11,185 @@ redirect_from:
+   **Using Google Container Engine?** Skip to the next step. GCE doesn't have any cluster options.
 
-2. In the Nodes section, click on **Add a new node** to select which type of node you want to add. You can configure a new node template to launch nodes or select from an existing node template (i.e. if you had previously launched a node). When launching any new nodes, a node template is saved with your configuration to allow you to re-use this configuration for adding additional nodes.
+   ![choose-cluster-options]
 
-3. After selecting your node(s) to be created, select which roles you want the node(s) to act in the Rancher managed Kubernetes cluster.
+5. **For those using Google Container Engine or Azure Container Service:**
 
-    Select from the following roles:
+     Complete the form asking for account information. The form includes links to instructions detailing how to obtain this info.
 
-    * **etcd** -- On this node, `etcd` is launched. Etcd is a distributed reliable key-value store which stores all Kubernetes state. We recommend running 1, 3, or 5 nodes with the etcd role.
-    * **management** -- On this node, master components will run (`kube-api`, `kube-scheduler`, `kube-controller`) as well as `kubelet` and `kubeproxy`. These nodes are used to help manage the Kubernetes cluster as well as where your applications (i.e. pods) can be launched.
-    * **worker node** -- On these nodes, only worker components (`kubelet`, `kubeproxy`, `nginx-proxy`) are launched and these nodes will only have your applications (i.e. pods) running.
+	 ![gce-azure-instructions]
 
-4. Finish creating your cluster by clicking **Create**. This process might take a few minutes to complete. Once your cluster is ready, you can view its status on the Clusters page. Once the cluster is active, you can start adding pods into your namespace.
+     **Using one of the other options?** This step doesn't apply to you. Skip to the next step.
 
-### Importing Kubernetes Clusters
+6. Add at least one **Node Pool**.
+
+   A *Node Pool* is a group of nodes that are configured identically. Your cluster can contain as many node pools as you'd like. Each object in the **Node Pools** grid represents a single node configuration. You can use the node pool to choose the number (i.e. **Count**) of nodes running a given configuration (i.e. **Template**).
+
+   > **Exception:** The instructions below don't apply to Azure Container Service. Node creation for Azure is a little different. Instead of accessing a template through an API, you'll have to complete the **Nodes** form, which  customizes nodes directly from Rancher.
+
+   1. Enter a **Node Prefix**. When the cluster is created, each node in the pool is named after the prefix. An incremented number is appended to each node.
+   2. Enter the node **Count** for the pool.
+   3. Click **Add Node Template**. A node template is just the a virtual machine configuration you're using to create a docker image.
```

Comment:
A node template is used to ultimately create a virtual machine/compute node, not a docker image.